README.yaml

```diff
@@ -238,6 +238,18 @@ examples: |-
   exit
   ```
 
+  ### Troubleshooting Package Addition
+  Here are some solutions to several common problems that may occur when adding a new package:
+  
+  1. <details><summary>When adding a new binary, the `make builder TARGETS=readme` command fails, claiming it can't find the binary file, even though it is in the expected place.</summary> 
```

Comment:
This error should not be triggered by making `readme`. It should only happen on `make -C vendor/<app> apk`.

Reply:
fixed by #925